config/locales/dates/dates.en.yml

```diff
@@ -92,6 +92,8 @@ en:
      long_date_time: "%A %d %B %Y at %I:%M %p"
      from_long_date_time: "From %A %d %B %Y at %I:%M %p"
      until_long_date_time: "To %A %d %B %Y at %I:%M %p"
+     from_long_date: "From %A %d %B %Y"
+     until_long_date: "To %A %d %B %Y"
```

Comment:
Very cool! Thanks for defining it in all the locales!
Next time you can just define the `en` locale since we have volunteers translating everything in the other languages through crowdin. Thank you anyway